runtime/interpreter/value.go

```diff
@@ -984,6 +984,11 @@ func (v CharacterValue) GetMember(interpreter *Interpreter, _ LocationRange, nam
 				)
 			},
 		)
+
+	case sema.CharacterTypeUtf8FieldName:
+		b := []byte(v)
+		common.UseMemory(interpreter, common.NewBytesMemoryUsage(len(b)))
+		return ByteSliceToByteArrayValue(interpreter, b)
```

Comment:
The allocation has to be metered before:

```suggestion
		common.UseMemory(interpreter, common.NewBytesMemoryUsage(len(v)))
		return ByteSliceToByteArrayValue(interpreter, []byte(v))
```